From 3b308c56c0fb588294841b784bb1a449e3832ef1 Mon Sep 17 00:00:00 2001
From: Richard Zhu <hongxing.zhu@nxp.com>
Date: Wed, 15 Feb 2017 16:45:07 +0800
Subject: [PATCH 1517/5242] MLK-13981-1 ARM: imx: add the virtual tty rpmsg
 channel

commit  78f24ac62b121e4e6374bb0f35d9852f7b593136 from
https://source.codeaurora.org/external/imx/linux-imx.git

add the virtual tty rpmsg channel, thus the message can
be transferred between A core and M core by rpmsg

Signed-off-by: Richard Zhu <hongxing.zhu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp-evk.dts |    5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx7ulp-evk.dts b/arch/arm/boot/dts/imx7ulp-evk.dts
index bd1dc02..493c6de 100644
--- a/arch/arm/boot/dts/imx7ulp-evk.dts
+++ b/arch/arm/boot/dts/imx7ulp-evk.dts
@@ -435,9 +435,10 @@
 	 * --0x9FF10000~0x9FF1FFFF: pmic
 	 * --0x9FF20000~0x9FF2FFFF: pm
 	 * --0x9FF30000~0x9FF3FFFF: audio
+	 * --0x9FF40000~0x9FF4FFFF: virtual tty
 	 */
-	vdev-nums = <4>;
-	reg = <0x9FF00000 0x40000>;
+	vdev-nums = <5>;
+	reg = <0x9FF00000 0x50000>;
 	status = "okay";
 };
 
-- 
1.7.9.5

